pontoon/contributors/templates/contributors/widgets/notifications_menu.html

```diff
@@ -76,7 +76,9 @@
           </span>
 
           <span class="verb">
+            {% if notification.verb != 'ignore' %}
```

Comment:
We should instead add another `or` in line 48, because we want that condition to be true.

```
or notification.verb == "has reviewed suggestions" or notification.verb == "ignore"
```